### PR TITLE
[#673] fix the healthCheckInterval value is not displayed correctly s…

### PR DESCRIPTION
**Pull request by david6969xin**
…ometimes

### CODE REVIEW

File: spring-cloud-huawei-common/src/main/java/com/huaweicloud/common/transport/DiscoveryBootstrapProperties.java [hunk 1]
```diff
@@ -171,7 +174,11 @@ public int getHealthCheckInterval() {
   }
 
   public void setHealthCheckInterval(int healthCheckInterval) {
-    this.healthCheckInterval = healthCheckInterval;
+    int healthCheckIntervalMillis = (int) TimeUnit.SECONDS.toMillis(healthCheckInterval);
+    if (Math.min(healthCheckIntervalMillis, ServiceCenterRegistration.MAX_INTERVAL) ==
+        Math.max(healthCheckIntervalMillis, ServiceCenterRegistration.MIN_INTERVAL)) {
+      this.healthCheckInterval = healthCheckIntervalMillis;
```
Comment (liubao68) [inline spring-cloud-huawei-common/src/main/java/com/huaweicloud/common/transport/DiscoveryBootstrapProperties.java:180]: `healthCheckInterval` 是秒，不能改变。 

这里直接增加两个常量即可，不用使用 ServiceCenterRegistration 的常量。

**Comment by david6969xin** [comment]
not fix, this situation will hardly occur in practical application scenarios.

File: spring-cloud-huawei-common/src/main/java/com/huaweicloud/common/transport/DiscoveryBootstrapProperties.java [hunk 2]
```diff
@@ -171,7 +176,10 @@ public int getHealthCheckInterval() {
   }
 
   public void setHealthCheckInterval(int healthCheckInterval) {
-    this.healthCheckInterval = healthCheckInterval;
+    if (Math.min(healthCheckInterval, HEALTH_CHECK_MAX_INTERVAL) ==
```
Comment (liubao68) [inline spring-cloud-huawei-common/src/main/java/com/huaweicloud/common/transport/DiscoveryBootstrapProperties.java:179]: Why not `HEALTH_CHECK_MIN_INTERVAL<=healthCheckInterval && HEALTH_CHECK_MAX_INTERVAL>= healthCheckInterval ` ?

Comment (david6969xin) [reply spring-cloud-huawei-common/src/main/java/com/huaweicloud/common/transport/DiscoveryBootstrapProperties.java]: OK

**Review by liubao68 — APPROVED** [review]

**Pull request merged**